webui/react/src/pages/DesignKit.tsx

```diff
@@ -202,38 +218,190 @@ const DropdownsSection: React.FC = () => {
         <strong>Accessibility</strong>
         <ul>
           <li>
-            ComboBox dropdowns render in their own layer by default to ensure they are not clipped
-            by containers with overflow: hidden or overflow: scroll. This causes extra difficulty
-            for people who use screen readers, so we recommend rendering the ComboBox options
-            dropdown inline unless they are in overflow containers.
+            Select dropdowns render in their own layer by default to ensure they are not clipped by
+            containers with overflow: hidden or overflow: scroll. This causes extra difficulty for
+            people who use screen readers, so we recommend rendering the ComboBox options dropdown
+            inline unless they are in overflow containers.
           </li>
         </ul>
         <strong>Truncation</strong>
         <ul>
           <li>
-            By default, the ComboBox truncates option text instead of wrapping to a new line.
-            Because this can lose meaningful information, it is recommended to adjust styles to wrap
-            the option text.
+            By default, the Select truncates option text instead of wrapping to a new line. Because
+            this can lose meaningful information, it is recommended to adjust styles to wrap the
+            option text.
           </li>
         </ul>
       </AntDCard>
       <AntDCard title="Usage">
-        <strong>Default dropdown</strong>
-        <SelectFilter
-          defaultValue={1}
+        <strong>Default Select</strong>
+        <Select
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+        />
+        <strong>Variations</strong>
+        <strong>Select with default value</strong>
+        <Select
+          defaultValue={2}
           options={[
             { label: 'Option 1', value: 1 },
             { label: 'Option 2', value: 2 },
             { label: 'Option 3', value: 3 },
           ]}
         />
-        <strong>Disabled dropdown</strong>
-        <SelectFilter
+        <strong>Select with placement</strong>
```

Comment:
the lables are too short to see the difference of `placement`. i think we need longer label insetad of `Option {number}`

Reply:
We opted to get rid of placement @keita-determined!